controller_manager/test/test_controller_failed_init/test_controller_failed_init.cpp

```diff
@@ -31,14 +31,6 @@ TestControllerFailedInit::on_init()
   return rclcpp_lifecycle::node_interfaces::LifecycleNodeInterface::CallbackReturn::FAILURE;
 }
 
-controller_interface::return_type TestControllerFailedInit::init(
-  const std::string & /* controller_name */, const std::string & /* urdf */,
-  unsigned int /*cm_update_rate*/, const std::string & /*node_namespace*/,
-  const rclcpp::NodeOptions & /*node_options*/)
-{
-  return controller_interface::return_type::ERROR;
-}
-
```

Comment:
Shouldn't we still keep this test?

What I understand, you are here not moving the node options only, but changing how init is provoking error in this case. It could be that I am missing a detail or something.

Reply:
Ideally, you don't need to have this as the same behavior will be achieved as the main `init` method in the base class will call the `get_node_options` method